### DISCUSSION TITLE
Convert multiple TIF to a single DFS2

### DISCUSSION THREAD

**Discussion by moinabyssinia** [DHI/mikeio#498] [Q&A] [answered]
I have multiple TIF files (rasters) and I would like to convert them to a single DFS2 file. Is there a way to do that using MIKEIO scripts?
I have exhausted the MIKE Zero and MIKE SHE toolbox functionalities.

Thanks,
Michael

**Comment by ecomodeller** [comment, marked as answer]
Yes, as long as you can get data in the form of a NumPy array, and know the coordinate reference system, you should be able to.

Take a look at [this notebook](https://dhi.github.io/mikeio-examples/notebooks/Dfs2_-_Global_Forecasting_System.html) for some inspiration, where we convert NetCDF  -> Dfs2. (to read TIF files in Python, take a look at [rasterio](https://rasterio.readthedocs.io/en/latest/quickstart.html))
